week-5/Homework/mandatory/3-project/js/main.js

```diff
@@ -0,0 +1,29 @@
+let btnBlue = document.querySelector("#blueBtn");
+btnBlue.addEventListener("click", ChangeColor);
+function ChangeColor() {
+  document.querySelector(".jumbotron").style.background = "#588FBD";
+  document.querySelector(".buttons .btn-primary").style.background = "#FFA500";
+  document.querySelector(".buttons .btn-secondary").style.background = "black";
+  document.querySelector(".buttons .btn-secondary").style.color = "white";
+}
+let orangeBtn = document.querySelector("#orangeBtn");
+orangeBtn.addEventListener("click", ChangeColor1);
+function ChangeColor1() {
+  document.querySelector(".jumbotron").style.background = "#f0ad4e";
+  document.querySelector(".buttons .btn-primary").style.background = "#5751fd";
+  document.querySelector(".buttons .btn-secondary").style.background =
+    "#31b0d5";
+  document.querySelector(".buttons .btn-secondary").style.color = "white";
+}
+let grnBtn = document.querySelector("#greenBtn");
+grnBtn.addEventListener("click", ChangeColor2);
+function ChangeColor2() {
+  document.querySelector(".jumbotron").style.background = "#87ca8a";
+  document.querySelector(".buttons .btn-primary").style.background = "black";
+  document.querySelector(".buttons .btn-secondary").style.background =
```

Comment:
💫It all works fine! 

How about storing all the elements in variables that you can then reuse.
That way, you won't have to write the same code over and over again.
eg 
```
let jumbotron = document.querySelector(".jumbotron");
jumbotron.style.background="white";
```
This will keep you code cleaner and more readable 😉

Reply:
100% right. Will pay more attention to it on all other projects